docs/Contributing.asciidoc

```diff
@@ -130,8 +130,9 @@ other modern web frameworks like Sinatra and that have nice and comprehensive
 documentation available at its http://mojolicio.us[home page].
 
 In addition to Mojolicious and its dependencies, several other CPAN modules are
-required by the openQA package. For a full list of hard dependencies, see the
-file `cpanfile` at the root of the openQA repository.
+required by the openQA package. You can find the dependencies in form of
```

Comment:
"the dependencies" is ambiguous as cpanfile does not state *all* dependencies, dependencies.yaml does though, right?

Maybe it's better to delete this block of the section further down

Reply:
What do you mean with "delete this block of the section further down"?

Reply:
Sorry, I meant: "Delete this block in favor of the section further down"